src/_posts/2018/2018-09-17-lessons-in-signage.md

```diff
@@ -23,7 +23,7 @@ We got a lot of stuff right off the bat:
 
 But this was a first attempt, and inevitably I didn't get it all right -- for the benefit of my future self, and anybody else making venue signage, here's a few of the things I've learnt this year.
 
-{% tweet "https://twitter.com/anabalica/status/1040947442735345664" %}
+{% tweet https://twitter.com/anabalica/status/1040947442735345664 %}
```

Comment:
One of the changes is that the plugin is a bit less tolerant now – it used to accept tweet URLs with/without quotes, because that's what my brain "expects" this to look like (I think I'd need to do this in Jinja2 maybe?).

But it's not actually required and I've only done it in two posts, so better to standardise on "never insert quotes".